### ISSUE TITLE
﻿Save Automatically Every Minutes (Autosave) just save scene

### ISSUE THREAD

**Issue by 2dvision** [opentoonz/opentoonz#414]
If Autosave to do "Save All", will be better. OT crashed many time and i lose my drawings.

**Comment by blurymind** [comment]
autosave should save everything/

Btw blender has a good approach to this - it creates myFile.blend1,myFile.blend2 while working on myFile.blend - these files are temporary and blender autosaves to them every few steps you make - that makes sure that no file gets corrupt during crash - only a copy of the file may get corrupt. When it crashes, blender allows you to recover the crashed file from myFile.blend2 even if you forgot to save.

Similar way is used in inkscape and kdenlive - both of which have great crash recovery.

**Comment by 2dvision** [comment]
>autosave should save everything

Yes and It's really necessary. I hope somebody fix that quickly.

**Comment by turtletooth** [comment]
Fixed in #474 

Along with:
![saveall](https://cloud.githubusercontent.com/assets/4576381/16175367/d0d8af78-35a7-11e6-822b-6c81b5b560af.png)

![savechanges](https://cloud.githubusercontent.com/assets/4576381/16175371/d3d98b16-35a7-11e6-9107-bcc1666776a9.png)

**Comment by blurymind** [comment]
@turtletooth autosave does not save all. The request here is to have autosave save all.
Not to have ctrl+s be save all.

Opentoonz latest nightly just crashed for me. I had autosave enabled and it kept saving.
After reopening the file - I found out that autosave did not save ANY of the new inking levels I created for myself:
![autosavenotsavingall](https://cloud.githubusercontent.com/assets/6495061/16309183/c784effc-395f-11e6-8805-51e8e560ead1.PNG)
So just like the original poster I have just lost all of my drawings, because 
preferences> general>save automatically every 5 minutes does NOT save the levels - it only saves the scene.
